cli/src/commands/edit.rs

```diff
@@ -21,10 +21,10 @@ use crate::cli_util::{CommandHelper, RevisionArg};
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
-/// Edit a commit in the working copy
+/// Sets the specified revision as the working-copy revision
 ///
-/// Puts the contents of a commit in the working copy for editing. Any changes
-/// you make in the working copy will update (amend) the commit.
+/// Note, it is generally recommended to instead to use `jj new` and `jj amend`. Creating
+/// a new revision enables being more intentional with respect to the existing revision.
```

Comment:
nit: I also have a tendency to write "Note", but I'm not sure it adds much. If you want to keep it, shouldn't it be "Note: It is..." or "Note that it is"?

Reply:
I think that as is is the same as using a colon in modern grammar? Though you wouldn't capitalize "it".

I'm mostly ambivalent on this, though I do think that saying "note" can help when the user is skimming, as it highlights that it's supplementary.

For the sake of avoiding indecision, I'll default to keeping "note", and switching to use the colon, unless you have further thoughts.

Reply:
> same as using a colon in modern grammar?

Oh, you're probably right. I'm old :) So feel free to ignore my suggestion.